### PR TITLE
Home/quote button

### PR DESCRIPTION
**Pull request by Sonny97**
Button about quote the stay, this button send to quote page.

### Desktop view:

image

### Mobile view: 

image

### CODE REVIEW

File: css/home.css [hunk 1]
```diff
@@ -161,6 +161,15 @@
...
 }
 
+.quote-container {
+  left: 0;
+  right: 0;
+  margin: auto;
+  text-align: center;
+  margin-bottom: 30px;
+  margin-top: 50px;
+}
```
Comment (camigomezdev) [inline css/home.css:167]: Are you sure all the properties are necessary?

Comment (Sonny97) [reply css/home.css:167]: changed, now look better?

Comment (camigomezdev) [reply css/home.css:167]: you could use 
```
    margin-top: 50px;
    text-align: center;
```
and for Desktop
```
    margin-top: 90px;
```
Why do you need `right` and `left` properties?

Comment (Sonny97) [reply css/home.css:167]: Done

**Review by jusk02 — APPROVED** [review]

**Review by Frankruiz90 — APPROVED** [review]

**Review by camigomezdev — APPROVED** [review]

**Pull request merged**